src/users/views.py

```diff
@@ -195,6 +198,51 @@ def get_context_data(self, **kwargs):
         context.update(**reviews_state()._asdict())
         return context
 
+def review_stages(request):
+    
+    review_stages_list = [
+        'Call for Proposals', 'First Round Review_1',
+        'First Round Review_2', 'Modification Stage',
+        'Second Round Review', 'Internal Decision',
+        'Announcement of Acceptance'
+    ]
+
+    if request.method == 'POST':
+
+        fmt = '%Y-%m-%d %H:%M:%S%z'
+        tz_selectd = pytz.timezone(request.POST['review_timezone'])
+        date_time_obj = datetime.datetime.strptime(
+            request.POST['proposals.disable.after'], '%Y-%m-%dT%H:%M:%S')
```

Comment:
I suppose this is the mismatched part. Acutally it will raise an exception without telling the user what went wrong. A better to do this is at least tell the user about the error format so that they know what happened.

Reply:
In the Google browser, **datetime-local** allows users to click on the date with a graphical button, but in  Safari browsers it is indeed to use text input, so there is the above problem.

Refer to your suggestion and use validation error. Its changes such as d579e69. When inputting a date format that cannot be parsed, the following prompt screen will be generated.

![image](https://user-images.githubusercontent.com/20749936/103173670-62d01e00-4897-11eb-86c8-a7653ade92d8.png)

Reply:
Yep, so here comes the validation part. If the value provided is invalid, it should return back to the frontend with some error messages so that the user know what should be fixed (usually some error message in red near the field, telling what's wrong and what should be expected)

Reply:
@mattwang44 also advised me used message.error instead of debug mode to remind users.
Done !
![image](https://user-images.githubusercontent.com/20749936/104792363-15c9d280-57d9-11eb-8f16-667f3fdb977c.png)